Sitemap/SitemapAwareDocumentVoter.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Symfony\Cmf\Bundle\SeoBundle\Sitemap;
+
+use Symfony\Cmf\Bundle\SeoBundle\SitemapAwareInterface;
+
+/**
+ * Implementing the SitemapAwareInterface will give the chance the let the document decide
+ * whether it wants to be displayed on sitemap or not.
+ *
+ * @author Maximilian Berghoff <Maximilian.Berghoff@mayflower.de>
+ */
+class SitemapAwareDocumentVoter implements VoterInterface
+{
+
+    /**
+     * {@inheritdoc}
+     */
+    public function exposeOnSitemap($content, $sitemap)
+    {
+        return $content instanceof SitemapAwareInterface && $content->isVisibleInSitemap();
```

Comment:
if we don't split that, every content that does not implement the interface will automatically be hidden from all sitemaps. imo if we want only such models, it would be the job of the provider to not load others in the first place.